sawtooth-sdk-transaction-processor/src/main/java/sawtooth/sdk/messaging/SendReceiveThread.java

```diff
@@ -140,8 +140,8 @@ public Message getMessage() {
      */
     DisconnectThread(final LinkedBlockingQueue<MessageWrapper> receiver,
         final ConcurrentHashMap<String, Future> hashMap) {
-      this.receiveQueue = receiveQueue;
-      this.futures = futures;
+      this.receiveQueue = SendReceiveThread.this.receiveQueue;
```

Comment:
This commit message needs a little tidying: summary line is too long

Reply:
fixed